test/new-e2e/tests/apm/vm_test.go

```diff
@@ -77,26 +94,18 @@ func (s *VMFakeintakeSuite) SetupSuite() {
 	h.MustExecute("sudo groupadd -f -r docker")
 	h.MustExecute("sudo usermod -a -G docker dd-agent")
 
-	// Create the /var/run/datadog directory and ensure
-	// permissions are correct so the agent can create
-	// unix sockets for the UDS transport
-	h.MustExecute("sudo mkdir -p /var/run/datadog")
-	h.MustExecute("sudo chown dd-agent:dd-agent /var/run/datadog")
-
 	// Restart the agent
 	h.MustExecute("sudo systemctl restart datadog-agent")
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Should you move also these steps up to the pulumi setup ?

Reply:
You mean move it in `WithUserData` ? I think at that point docker and the agent wouldn't be installed yet.